packages/geosearch-requester/rollup.config.ts

```diff
@@ -0,0 +1,19 @@
+import resolve from 'rollup-plugin-node-resolve';
```

Comment:
[`rollup-plugin-node-resolve`](https://github.com/rollup/rollup-plugin-node-resolve) was needed to get Rollup to properly find `@justfixnyc/util` and bundle it into the output.

I looked at the output bundle and it is actually performing tree-shaking properly and pulling _only_ `getHTMLElement()` from `@justfixnyc/util` without the intermediary `index.js` re-export, which is nice.